pkgs/top-level/all-packages.nix

```diff
@@ -12419,7 +12419,7 @@ with pkgs;
       num =
         if (with stdenv.targetPlatform; isVc4 || libc == "relibc") then 6
         else if (stdenv.targetPlatform.isAarch64 && stdenv.isDarwin) then 11
-        else if (stdenv.targetPlatform.isx86_64) then 11
+        else if (stdenv.targetPlatform.isx86_64 || stdenv.targetPlatform.isRiscV) then 11
         else if stdenv.targetPlatform.isAarch64 then 9
         else 10;
```

Comment:
It reads `else 10`, I believe, the same line is in staging.

Reply:
I think what is meant is that most target platforms are already using 11, with this change.